apps/condo/migrations/20220728132452-0157_auto_20220728_0824.js

```diff
@@ -0,0 +1,89 @@
+
+exports.up = async (knex) => {
+    await knex.raw(`
+    BEGIN;
+        --
+        -- Rename model "Device" to "RemoteClient"
+        --
+        ALTER TABLE "Device" RENAME TO "RemoteClient";
+        
+        --
+        -- Add "appId" field and update it's values
+        --
+        ALTER TABLE "RemoteClient" ADD COLUMN "appId" text DEFAULT 'unknown' NOT NULL;
+        UPDATE "RemoteClient" SET "appId" = (CASE WHEN LENGTH(SUBSTRING("deviceId", 38)) > 0 THEN SUBSTRING("deviceId", 38) ELSE gen_random_uuid()::text END);
```

Comment:
100% will fail during migration process because of timeout

Reply:
can we just drop that devices?

Reply:
It's better to add new fields and then rename ` "Device" RENAME TO "RemoteClient";`